lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -213,4 +216,16 @@ sub get_dns_server_setup {
     return YaST::DNSServer::DNSServerSetupController->new();
 }
 
+sub get_kdump_startup {
+    return YaST::Kdump::StartUpPage->new({app => YuiRestClient::get_app()});
+}
+
+sub get_kdump_firmware {
```

Comment:
would be more clear: `get_kdump_fadump_startup` so we follow the name of the package.